packages/calcite-components/src/components/block/block.scss

```
@@ -11,6 +11,9 @@
  * @prop --calcite-block-heading-text-color-press: When the component is `expanded`, specifies the `heading` text color.
  * @prop --calcite-block-padding: [Deprecated] Specifies the padding of the component's `default` slot.
  * @prop --calcite-block-text-color: Specifies the component's text color.
+ * @prop --calcite-block-description-text-color: Specifies the component's description text color.
+ * @prop --calcite-block-icon-color: Specifies the component's icon color.
+ * @prop --calcite-block-icon-color-hover: Specifies the component's icon color when hovered.
```

Sidebar: Since this applies to toggle icon only, should we add that in the doc?

cc @geospatialem

---

It looks like the icon color is assigned to the `collapsable`, `iconStart` and `iconEnd`, should we consider adding the context? 

Would we allow more individual customization for all three?

```suggestion
* @prop --calcite-block-icon-color: Specifies the component's `collapsible` icon, `iconStart`. and `iconEnd` color.
 * @prop --calcite-block-icon-color-hover: Specifies the component's `collapsible` icon,  `iconStart`, and `iconEnd` color when hovered.
```